weblab_cg/__init__.py

```diff
@@ -29,7 +29,20 @@
     load_template,
 )
 
-from ._printer import (     # noqa
+from ._generator_chaste_cg import (   # noqa
```

Comment:
Now we have more stuff here, I'm not convinced by this style of having 'private' modules and then exposing names publicly via `__init__.py`. Why not let users import the modules they actually care about?

Probably don't fix this in this PR though; create an issue for it?

Reply:
Sounds like something we should discuss, but not for this PR